src/reviews/views.py

```diff
@@ -210,6 +210,24 @@ def get_object(self):
             )
         except Review.DoesNotExist:
             review = None
+
+        if review is None and settings.REVIEWS_STAGE == 2:
+            # If the reviewer reviewed the same proposal in the first stage,
+            # initialise the form with previous vote and comment as default values.
+            try:
+                previous_review = Review.objects.get(
+                    proposal=self.proposal,
+                    reviewer=self.request.user,
+                    stage=1,
+                )
```

Comment:
Might be better to use `> 1` and `stage=settings.REVIEWS_STAGE - 1` to make this more future-proof? THis would make sense if there is a third round IMO.

Reply:
Also I kind of wonder whether it is better to do this with `initial` (like you did previously in forms) instead.

Reply:
This is the scenario where the form looks like updating but is actually creating, I don't know if there's a good pattern to follow in Django.  For me I prefer my previous solution because semantically this is a part of the form, and I kinda like the way of setting the initial values because there's the word "initial" in there which makes semantic sense.  Maybe there's a better place to put it in views but this is the best I've found.